dbt_project.yml

```diff
@@ -1,5 +1,5 @@
 name: 'the_tuva_project'
-version: '0.4.1'
+version: '0.5.0'
```

Comment:
We just went to 0.4 recently. I think when we originally talked about going to 0.4 I thought it would include more of the changes needed for clinical support. Should we just skip on to 0.5 or stay in 0.4 a little while longer? I definitely realize this is a big/breaking change.

Reply:
I think we should increment whenever there is a major change to functionality.  I think patches say that the code still works the same but there are areas that have been tweaked.  When the minor changes it signals (to me) that there is a breaking change or the code works differently.  Also, it's a way to mark a major milestone in the project.